packages/components/checkbox/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as checkbox } from './Checkbox';
```

Comment:
A component name must always be uppercase.
Also, you forgot to export CheckbocGroup.